core/state/statedb.go

```diff
@@ -474,6 +474,17 @@ func (s *StateDB) Suicide(addr common.Address) bool {
 	return true
 }
 
+func (s *StateDB) Selfdestruct6780(addr common.Address) {
+	stateObject := s.getStateObject(addr)
+	if stateObject == nil {
+		return
```

Comment:
It also looks like `getStateObject` will never return `nil` here because the currently executing address cannot be a deleted state object.